pymc/tests/logprob/test_transforms.py

```diff
@@ -807,6 +807,36 @@ def test_reciprocal_rv_transform(numerator):
     )
 
 
+def test_sqr_transform():
+    # The square of a unit normal is a chi-square with 1 df
+    x_rv = at.random.normal(0, 1, size=(3,)) ** 2
+    x_rv.name = "x"
+
+    x_vv = x_rv.clone()
+    x_logp_fn = pytensor.function([x_vv], joint_logprob({x_rv: x_vv}, sum=False))
+
+    x_test_val = np.r_[0.5, 1, 2.5]
+    assert np.allclose(
+        x_logp_fn(x_test_val),
+        sp.stats.chi2(df=1).logpdf(x_test_val),
+    )
```

Comment:
It's not obvious from the test alone, but the `joint_logprob` here only works because the `measurable_sqrt_sqr_to_power` rewrite and `PowerTransform(RVTransform)` were implemented, right?

Maybe you can add this info in the docstring and assert that the `x_logp_fn` has the rewritten graph structure?

(Same comment applies to `test_sqrt_transform`.)

Reply:
It shouldn't matter how the logp was achieved only that we got it and it's correct.